Core/scripts/dirac-install.py

```diff
@@ -2476,11 +2462,6 @@ def createBashrcForDiracOS():
       lines.extend(['# new OpenSSL version require OPENSSL_CONF to point to some accessible location',
                     'export OPENSSL_CONF=/tmp'])
 
-      # gfal2 requires some environment variables to be set
-      # Note: eventually this line should disappear as already set by diracosrc
-      lines.extend(['# Gfal2 configuration and plugins',
-                    'export GFAL_CONFIG_DIR=$DIRACOS/etc/gfal2.d',
-                    'export  GFAL_PLUGIN_DIR=$DIRACOS/usr/lib64/gfal2-plugins/'])
```

Comment:
Suppose that this version of dirac-install will be used for installing DIRAC v6r21. What would happen?

Reply:
this is only for `diracos` so no risk I would say